src/assets.ts

```diff
@@ -488,7 +481,18 @@ function addLaunchJsonIfNecessary(generator: AssetGenerator, operations: Operati
 
         const isWebProject = generator.hasWebServerDependency();
         const launchJson = generator.createLaunchJson(isWebProject);
-        const launchJsonText = JSON.stringify(launchJson, null, '    ');
+
+        const configurationsMassaged = JSON.stringify(launchJson, null, '   ').split('\n').map(line => '   ' + line).join('\n').trim();
+
+        const launchJsonText = [
```

Comment:
Why not just use a template string so you don't have to use an array and join it at the end?

E.g.

```TypeScript
const launchJsonText = `
{
    // Use IntelliSense to find out which attributes exist for C# debugging
    // Use hover for the description of the existing attributes
    // For further information visit https://go.microsoft.com/fwlink/?linkid=830387
    "version": "0.2.0",
    "configurations": ${configurationsMassaged}
}`;
```